data/index.html

```diff
@@ -1,29 +1,145 @@
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
+ <link rel="stylesheet" href="index.css">
 </head>
 
+
 <body>
-  <header>
-    <h1>Solar System</h1>
-  </header>
-  
-  <div class="main">
-    <div class="menu">
-      <ul class="list-group">
-        <li><a href="mercury.html" class="list-group-item">Mercury</a></li>
-        <li><a href="venus.html" class="list-group-item">Venus</a></li>
-        <li><a href="earth.html" class="list-group-item">Earth</a></li>
-        <li><a href="mars.html" class="list-group-item">Mars</a></li>
-        <li><a href="jupiter.html" class="list-group-item">Jupiter</a></li>
-        <li><a href="saturn.html" class="list-group-item">Saturn</a></li>
-        <li><a href="uranus.html" class="list-group-item">Uranus</a></li>
-        <li><a href="neptune.html" class="list-group-item">Neptune</a></li>
-      </ul>
-    </div>
-  </div>
+ <header>
+   <nav class="navbar">
+     <div class="logo">
+       <i class="fas fa-sun"></i> Solar System
+     </div>
+     <ul class="nav-links">
+       <li><a href="#about">About</a></li>
+       <li><a href="#planets">Planets</a></li>
+       <li><a href="#facts">Fact</a></li>
+     </ul>
+     <div class="hamburger" aria-label="Toggle navigation" role="button" tabindex="0">
+       <i class="fas fa-bars"></i>
+     </div>
+   </nav>
+ </header>
+
+
+ <main>
+   <section class="hero">
+     <div class="hero-overlay">
+       <h1>Explore the Solar System</h1>
+       <p>Discover the wonders of our celestial neighborhood without the need for images.</p>
+       <a href="#planets" class="btn">Get Started</a>
+     </div>
+   </section>
+
+
+   <section id="about" class="about-section">
+     <h2>About the Solar System</h2>
+     <p>
+       The Solar System comprises the Sun and all celestial bodies bound to it by gravity, including eight planets, their moons, dwarf planets, and countless small objects like asteroids and comets.
+     </p>
+   </section>
+
+
+   <section id="planets" class="planets-list">
+     <h2>Planets</h2>
+     <ul>
+       <li><a href="mercury.html">Mercury</a></li>
+       <li><a href="venus.html">Venus</a></li>
+       <li><a href="earth.html">Earth</a></li>
+       <li><a href="mars.html">Mars</a></li>
+       <li><a href="jupiter.html">Jupiter</a></li>
+       <li><a href="saturn.html">Saturn</a></li>
+       <li><a href="uranus.html">Uranus</a></li>
+       <li><a href="neptune.html">Neptune</a></li>
+     </ul>
+   </section>
+
+
+   <section id="facts" class="facts-section">
+     <h2>Solar System Fact</h2>
+     <div id="fact-container">
+       <!-- to display random facts -->
+     </div>
+   </section>
+ </main>
+
+
+ <footer>
+   <p>&copy; 2024 Solar System Explorer. All rights reserved.</p>
```

Comment:
We can't accept content that is copyright and without license, which is what "all rights reserved" means.  This is a blocker.  Seek the legal consent of the copyright owner and obtain a license compatible with our project, such as Creative Commons, or remove the content.